### PR TITLE
1. Replace Express with FastAPI 

### PR DESCRIPTION
**Pull request by harsweet**
Does away with the ExpressJS stuff and adds FastAPI with one endpoint for getting the calendar json

Screen Shot 2022-11-17 at 9 08 49 AM

Screen Shot 2022-11-17 at 9 08 12 AM

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import uvicorn
+from fastapi import FastAPI
+from handlers import calendarHandler
+
+app = FastAPI()
+
+@app.get("/calendarJson")
```
Comment (cloudyyoung) [inline server/app.py:7]: Feel like kabab-case is better for url

**Review by cloudyyoung — APPROVED** [review]
LGTM!

File: server/app.py (hunk 1 above)
Comment (tim-macphail) [reply server/app.py:7]: Agree

**Review by tim-macphail — APPROVED** [review]

File: server/app.py (hunk 1 above)
Comment (tim-macphail) [reply server/app.py:8]: We'll leave this as a good-first-issue for someone else to fix later :)

**Pull request merged**